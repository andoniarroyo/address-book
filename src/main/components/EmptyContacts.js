// @flow
// external modules
import React from 'react';
// styles
import '../styles/components/emptyContact.scss';

/**
 * The empty contact render method
 * @returns {React$Element<any>} HTML markup for the component
 * @public
 */
const EmptyContacts = () => (
  <article className="empty-container">
    Your address book is empty so far...
    Why not create a new contact?
  </article>
);

export default EmptyContacts;
