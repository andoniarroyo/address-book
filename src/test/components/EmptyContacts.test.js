// external modules
import React from 'react';
import renderer from 'react-test-renderer';
// internal modules
import EmptyContacts from '../../main/components/EmptyContacts';

describe('<EmptyContacts />', () => {
  it('renders the component', () => {
    const tree = renderer.create(<EmptyContacts />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
