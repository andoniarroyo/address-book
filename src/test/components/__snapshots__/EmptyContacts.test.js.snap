// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptyContacts /> renders the component 1`] = `
<article
  className="empty-container"
>
  Your address book is empty so far... Why not create a new contact?
</article>
`;
